identity/webapp/src/frontend/components/Layout.style.ts

```diff
@@ -20,6 +33,24 @@ export const Header = styled(Space)`
 
 export const Title = styled.h1.attrs({ className: font('wb', 0) })``;
 
+type SectionHeadingProps = {
+  as?: 'h1' | 'h2' | 'h3' | 'h4' | 'h5' | 'h6';
+  removeBottomPadding?: boolean;
+};
+
+export const SectionHeading = styled(Space).attrs<SectionHeadingProps>(
+  props => ({
+    as: props.as || 'h2',
+    v: {
+      size: 'm',
+      properties: props.removeBottomPadding ? [] : ['padding-bottom'],
```

Comment:
Is there a reason we don't remove this from `./account`? If anything there's definitely more times where this is on, so maybe we should make it the default?

Reply:
It's because on the account page the headings appear outside of the white container panels, whereas everywhere else they are inside.

![Screenshot 2021-12-01 at 10 56 49](https://user-images.githubusercontent.com/6051896/144222550-04e0800d-cf07-4f23-b173-7b1d5e1ab577.png)

![Screenshot 2021-12-01 at 10 55 21](https://user-images.githubusercontent.com/6051896/144222496-8c12c03e-add2-4c08-bb1d-ff84c50e43e0.png)

I'll switch which is the default.